src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -380,6 +380,8 @@ def electricity(scenario):
             "wind_offshore": read_costs(costs, "offwind", "VOM"),
             "wind_onshore": read_costs(costs, "onwind", "VOM"),
             "solar": read_costs(costs, "solar", "VOM"),
+            "run_of_river": 0,
+            "reservoir": 0,
```

Comment:
Where are these values coming from? Could you provide a source?

Reply:
I checked the last version of [PyPSA technology-data](https://github.com/PyPSA/technology-data/tree/master/outputs) and it is still the case, that there are no available VOM data for __ror__ and __reservoir__. Since it is a common assumption to say that the marginal costs of these technologies are 0 or near 0, I included that value in the scenario_parameters script.

I could add i.e this [report](https://www.aemo.com.au/-/media/Files/Electricity/NEM/Planning_and_Forecasting/Inputs-Assumptions-Methodologies/2019/Report-Pumped-Hydro-Cost-Modelling.pdf) which mentioned for hydro generation: "all operations and maintenance costs might be categorized as fixed rather than variable, as the cost of operation is not directly tied to the amount of energy produced"